bayesian_changepoint_detection_experiment/03_changepoint_exploration_experiment.py

```diff
@@ -15,7 +15,7 @@
 
 # # Dive into time periods of change
 
-# This notebook is designed to further examine individual changepoints that occurred in [01_change_point_detection.ipynb](01_change_point_detection.ipynb).
+# This notebook is designed to further examine individual changepoints that occurred in [01_change_point_detection.ipynb](01_change_point_detection.ipynb) and [02_changepoint_detection_comparison](02_changepoint_detection_comparison).
```

Comment:
Are you specifically trying to compare different quantifications of change - frequency_ratio vs ratio_metric? I think I got a little confused as to the difference between these two notebooks

Reply:
>Are you specifically trying to compare different quantifications of change - frequency_ratio vs ratio_metric? I think I got a little confused as to the difference between these two notebooks 

Ah my bad. Notebook 02 is designed to compare the bayesian approach and the CUSUM approach in terms of can they detect a changepoint where one expects a changepoint to occur (hence the pandemic positive control). Notebook 3 is following up on Notebook 2's conclusion by actually uncovering the specific changes that were detected. Hence the dot plots next to the words (can only tell changes by looking at the top X most similar words for each year).